lib/handler/retry-handler.js

```diff
@@ -36,6 +35,7 @@ class RetryHandler {
     this.aborted = false
     this.retryOpts = {
       retry: retryFn ?? RetryHandler[kRetryHandlerDefaultRetry],
+      ifMatch: ifMatch ?? true,
```

Comment:
Why option?

Reply:
To allow disable the `if-match` if not interested into using it; just to give the opportunity to disable the request conditional.